database/migrations/20190219185900_foreignkey.js

```diff
@@ -0,0 +1,9 @@
+exports.up = function(knex, Promise) {
+  return knex.schema.table('users', table => {
+    table.integer('contacts_id').references('contacts_id').inTable('contacts').notNullable();
+  });
+};
+
+exports.down = function(knex, Promise) {
+  return knex.schema.dropTable('users');
```

Comment:
For this migration you shouldn't be dropping the table, that's not the opposite of what the up function is doing. It should be dropping that column you just created. Nothing you can do now for this though but keep note of what you should be doing in a down function.